grails-app/services/org/pih/warehouse/replenishment/ReplenishmentService.groovy

```diff
@@ -7,18 +7,25 @@
  * the terms of this license.
  * You must not remove this notice, or any other, from this software.
  **/
-package replenishment
+package org.pih.warehouse.replenishment
```

Comment:
thank you. i just noticed this last night when i was browsing the code for something else.